src/modules/staking/staking.types.ts

```diff
@@ -0,0 +1,23 @@
+import { BigNumber } from "ethers";
+
+export enum StakeStatus {
+    NONSTAKING = 0,
+    STAKING = 1,
+    WITHDRAWING = 2,
+  }
+  
+  export type Service = {
+    /** organization ENS name */
+    org: string;
+    /** pool address */
+    pool: string;
+    /** provider address */
+    provider: string;
+  };
+  
+  export type Stake = {
+    amount: BigNumber;
+    depositStart: BigNumber;
+    depositEnd: BigNumber;
+    status: StakeStatus;
+  };
```

Comment:
new line, and formatting seems a little bit weird in this file (spaces before export keyword)